modules/sr/robot/output_frequency_limiter.py

```diff
@@ -0,0 +1,23 @@
+from controller import Robot
+
+# Allow up to two changes per second
+MAX_FREQUENCY = 2
```

Comment:
Should the frequency be an argument to the limiter?

Reply:
I thought about that but decided it would be easy enough to add that support when we needed it.